docs/components/llms/conceptual_guide.md

```diff
@@ -1,5 +1,19 @@
 # Conceptual Guide
```

Comment:
nit: I've been calling this `Key Concepts` in my docs updates instead of `Conceptual guide`, would be good to standardize on one.

Reply:
i like key concepts!